watertap/unit_models/reverse_osmosis_base.py

```diff
@@ -370,6 +370,64 @@ def eq_flux_mass(b, t, x, p, j):
                     - prop_perm.conc_mass_phase_comp[p, j]
                 )
 
+        if self.config.transport_model == "SKK":
+
+            self.eq_flux_mass.deactivate()
+
+            self.reflect_coeff = Var(
+                initialize=1,
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Reflection coefficient of the membrane",
+            )
+
+            self.alpha = Var(
+                initialize=1e-8,
+                domain=NonNegativeReals,
+                units=units_meta("time") * units_meta("length") ** -1,
+                doc="Alpha coefficient of the membrane",
+            )
+
+            @self.Constraint(
+                self.flowsheet().config.time, solute_set, doc="SKK alpha coeff."
+            )
+            def eq_alpha(b, t, j):
+                return b.alpha == (1 - b.reflect_coeff) / b.B_comp[t, j]
+
+            @self.Constraint(
+                self.flowsheet().config.time,
+                self.difference_elements,
+                self.config.property_package.phase_list,
+                self.config.property_package.component_list,
+                doc="Solvent and solute mass flux using SKK model",
+            )
+            def eq_flux_mass_skk(b, t, x, p, j):
```

Comment:
Just to echo earlier comments, I figured this constraint wouldn't need to be built anew. Rather, we just use this single constraint (rename it back to eq_flux_mass) for both transport models, so long as numerous tests don't fail due to unintuitive numerical issues.